src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Pages/TradingName.cshtml

```diff
@@ -0,0 +1,307 @@
+﻿@using SFA.DAS.AssessorService.ApplyTypes.Roatp
+@using SFA.DAS.AssessorService.Domain.Extensions
+@model SFA.DAS.AdminService.Web.ViewModels.Roatp.Gateway.TradingNamePageViewModel
+
+@{
+    var heading = "Trading name check";
+    var caption = "Organisation checks";
+
+    ViewBag.Title = $"Gateway - {heading}";
+    Layout = "_Layout";
+
+    var formGroupError = Model.ErrorMessages != null && Model.ErrorMessages.Any() ? "govuk-form-group--error" : "";
```

Comment:
Bit of a code smell here... you should be using ModelState.IsValid which is the design pattern used in most other places